test/socketpair_helper.py

```diff
@@ -1,5 +1,10 @@
 import socket
 
+try:
+    _CONNECT_ERROR = (BlockingIOError, InterruptedError)
+except:
```

Comment:
Sure, will be an additional `try` and `catch` as non-Windows platforms don't define `WinError`.